cmd/lotus-miner/proving.go

```diff
@@ -238,29 +249,51 @@ var provingDeadlinesCmd = &cli.Command{
 			}
 
 			sectors := uint64(0)
+			active := uint64(0)
 			faults := uint64(0)
+			live := uint64(0)
+			recovering := uint64(0)
 
-			for _, partition := range partitions {
+			for partIdx, partition := range partitions {
 				sc, err := partition.AllSectors.Count()
 				if err != nil {
 					return err
 				}
-
 				sectors += sc
 
-				fc, err := partition.FaultySectors.Count()
+				ac, err := partition.ActiveSectors.Count()
 				if err != nil {
 					return err
 				}
+				active += ac
 
+				fc, err := partition.FaultySectors.Count()
+				if err != nil {
+					return err
+				}
 				faults += fc
+
+				rc, err := partition.RecoveringSectors.Count()
+				if err != nil {
+					return err
+				}
+				recovering += rc
+
+				lc, err := partition.LiveSectors.Count()
+				if err != nil {
+					return err
+				}
+				live += lc
+				if cctx.IsSet("detailed") {
+					_, _ = fmt.Fprintf(tw, "%d\t%d\t%d (%d|%d|%d|%d)\t\n", dlIdx, partIdx, sc, fc, rc, lc, ac)
```

Comment:
I think we should just create tabbed columns for all the new data we're displaying.